### ISSUE TITLE
proxy and cache not recognized in 'check' when vulnerabilities are found

### ISSUE THREAD

**Issue by dhrmn** [pyupio/safety#233]
* safety version: 1.8.5
* Python version: 3.5.3
* Operating System: Debian stretch

### Description

When using the '--proxy' options with 'safety check', the command hangs/times out, trying to fetch the DB URL directly, ignoring the proxy. This happens whenever vulnerabilities are found.

Furthermore, the '--cache' option is also ignored.

### What I Did

```
safety check --proxy-host 192.168.1.1 --proxy-port 8888 --proxy-protocol http --cache
```
In firewalled environments, where the machine has no access to fetch the safety DB directly, this command hangs and times out after a while. The buggy behaviour is only triggered when vulnerable packages have been found.

**Comment by dhrmn** [comment]
Solved with pull request #234

**Comment by rafaelpivato** [comment]
Thanks for that fix!
